README.md

```diff
@@ -229,6 +229,37 @@ Specifies target architectures when using the multiplatform build CLI goal.
 
 *Default:* None
 
+#### wheelDependencies ####
+
+Optional set of wheel dependencies to retrieve from poetry cache. This allows previously cached external 
+wheel dependencies to be copied into a given target directory if it exists in poetry cache. This logic 
+depends on wheels to have first been cached by `cacheWheels` habushu-maven-plugin configuration and executes
```

Comment:
A: You should be able to add a link here to the actual `cacheWheels` description.

Reply:
On second thought, it might make more sense to link to the `install` goal [documentation](https://github.com/TechnologyBrewery/habushu?tab=readme-ov-file#install) since that's what caches the wheels.